packages/core/src/internal-components/overlay/overlay.stories.ts

```diff
@@ -87,6 +90,9 @@ export const interactive = () => {
       myOverlay.addEventListener('closeChange', () => {
         myOverlay.setAttribute('hidden', 'true');
       });
+      myOverlay.addEventListener('motionChange', (e: any) => {
+        console.log(e.detail);
```

Comment:
I assume the `console.log` here is for testing? Just checking that you weren't intending to do something additional in this event listener.

Reply:
Yeah. That's just for demo purposes.